src/config/constants.ts

```diff
@@ -15,10 +16,10 @@ export const BEAMER_ID = process.env.NEXT_PUBLIC_BEAMER_ID || ''
 
 // Wallets
 export const WC_BRIDGE = process.env.NEXT_PUBLIC_WC_BRIDGE || 'https://safe-walletconnect.gnosis.io/'
-export const FORTMATIC_KEY = process.env.NEXT_PUBLIC_FORTMATIC_KEY || ''
-export const PORTIS_KEY = process.env.NEXT_PUBLIC_PORTIS_KEY || ''
-export const TREZOR_APP_URL = 'gnosis-safe.io'
-export const TREZOR_EMAIL = 'safe@gnosis.io'
+export const TREZOR_APP_URL = 'app.safe.global'
```

Comment:
Have you tested that this works on staging? Maybe it should be dynamic.

Reply:
It's just for links I believe. As you can see `gnosis-safe.io` has been working fine all along.